Add information typing pages (#146)
mongodb/docs-meta: source/information-types/reference.txt

.. _meta-reference-prototype:

========================
Reference Page Prototype
========================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

A reference page provides quick information to experienced users of a product. 
Reference pages may describe the use of a method, operator, or database command. 
Information presented as a bulleted list or table is usually a reference page. 
Release notes are another good example of a reference page.

Usually, the title of a reference page is the name of the command, operator, or 
method it describes.

Definition
----------

A reference page begins with a brief introduction summarizing its contents, 
often called a short description. The short description explains a little about 
how the reference material is used, and may link to a concept and/or task page.

Syntax
------

> This is the sample syntax with the required parameters. 

Reviewer: ```suggestion
This section includes the sample syntax with the required parameters. 
```

        name: "<shard_name>"                                               
      }                                                                    
   )  

Command Fields
--------------

The command takes the following fields:

.. list-table::                                                            
   :header-rows: 1                                                         
   :widths: 20 20 10 60                                                       
                                                                           
   * - Field                                                               
                                                                           
     - Type 

     - Necessity                                                               
                                                                           
     - Description

   * - Parameter

     - String

     - Required

     - String

Behaviors
---------

Optional. 

Behaviors should only apply to the operator being described on this page. 

Examples
--------

Optional.

This section could contain examples of the most common usages. We would populate

> Reference and Task page examples using includes, so that the examples are written
> once, consistent throughout, and easier to update.

Reviewer: ```suggestion
Reference and Task page examples using includes, so that we write the examples
once, present them consistently throughout, and can update them easily.
```

> If an example is shared between task and reference, it should be written as an
> include.

Reviewer: ```suggestion
If an example is shared between task and reference, you should write it as an
include.
```
